src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capasitu = 16;
+    private float loadFactor = 0.75f;
```

Comment:
You have a magic number here - 0.75f. It's better to make it a constant with a meaningful name. For example, LOAD_FACTOR.